app/src/contexts/AuthContext.tsx

```diff
@@ -68,7 +68,7 @@ export function AuthProvider({ children }) {
           baseURL: `${process.env.REACT_APP_BACKEND_ADDRESS}`,
           headers: {
             "x-api-key": `${process.env.REACT_APP_API_KEY}`,
-            "session-token": sessionToken,
+            "session-token": sessionToken!,
```

Comment:
why the non-null assertion here? are you 100% sure that `sessionToken` will always be defined? if not, I would take this out

Reply:
For some reason, it had a red underline without it? I don't know why somethings show that error despite me not having changed anything